tspl.js

```diff
@@ -60,7 +55,9 @@ function tspl (t, opts = {}) {
     }
   }
 
-  return res
+  plan.end = end
+  plan.completed = plan
+  return plan
```

Comment:
Given past experiences with typescript-eslint and friends, can you make it a Thenable instead? Returning a promise would have type-sensitive checkers always ask it to be awaited.